exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/ApiControllerIntegrationTest.java

```diff
@@ -754,6 +760,53 @@ void getActualConfiguration(VertxTestContext context) {
         })));
   }
 
+  @Test
+  void getTime(VertxTestContext context) {
+    ZonedDateTime time = ZonedDateTime.of(2018, 1, 1, 0, 0, 0, 0, ZoneOffset.UTC);
+    when(qaService.getTime()).thenReturn(Optional.of(time));
+
+    String expectedStringTime = convertZdtToString(time);
+
+    get(TIME_PATH)
+        .send(context.succeeding(response -> context.verify(() -> {
+          assertThat(response.statusCode())
+              .isEqualTo(HTTP_OK);
+
+          String body = response.bodyAsString();
+          Object actualTime = JSON_SERIALIZER
+              .fromJson(body, new TypeToken<String>() {
```

Comment:
TypeToken can be replaced by just `String.class`.
But, is it ok to return raw string instead of json object? @dip56 WDYT?

Reply:
actualy nevermind, but response in `json` only for consistency